Misc/NEWS.d/next/Library/2023-07-12-03-04-45.gh-issue-106664.ZeUG78.rst

```diff
@@ -0,0 +1 @@
+Reduce overhead to add new selectors
```

Comment:
This paragraph is not helpful. My suggestion is:

```
:mod:`selectors`: Add `_SelectorMapping.get()` method and optimize `_SelectorMapping.__getitem__()`.
```

(I am not native English speaker. Please don't believe me about English sentense.)